docs/advanced_usage.rst

```diff
@@ -241,6 +241,46 @@ next step of the pipeline or ``None`` if the trace should be discarded::
 
 (see filters.py for other example implementations)
 
+.. _`Logs Injection`:
+
+Logs Injection
+--------------
+
+Datadog APM traces can be integrated with Logs by setting an opt-in flag and
+updating the log formatter used by an application. This feature enables the user
+to move from a trace to the specific log entry emitted within the same trace
+context.
+
+Before the trace information is injected into logs, the formatter has to be
+updated to include ``trace_id`` and ``span_id`` attributes from the log record.
+Furthermore, the integration with Logs occurs automatically as long as the
+format of logs is appended with ``- dd.trace_id=%(trace_id)s dd.span_id=%(span_id)s``.
+
+When using ``ddtrace-run``, set the environment variable
+``DD_LOGS_INJECTION=true``. If you prefer to manual instrumentation::
+
+    from ddtrace import patch_all; patch_all(logging=True)
+
+An example of logs injection with manual instrumentation::
```

Comment:
Can we have two distinct examples, maybe even with sub-titles.

1) Showing how to use `ddtrace-run` and set the format string
2) Showing how to instrument manually and set the format string


I just want to make sure these instructions are super clear.

Reply:
Yeah. The activation process is super simple, but since we have 2 steps (enable the "patch" AND change the formatter) it's very easy to think that `DD_LOGS_INJECTION=true` is enough to have it work. I want to be sure it's clear that the process is 2 steps whatever option you pick (`ddtrace-run` or manual instrumentation).